modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -230,6 +231,68 @@ const FrontendDataSet = ({
 		setItems(dataSetData.items);
 	}
 
+	useEffect(() => {
+		loadClientExtensions([
+			{
+				imports: initialFilters
+					?.filter((filter) => filter.cxFilterURL)
+					.map((filter) => ({
+						context: filter,
+						importDeclaration: `default from ${filter.cxFilterURL}`,
+					})),
```

Comment:
I like `importDeclaration`, but `imports` sounds a bit awkward, as `context` is unrelated to import. Maybe `cxDefinitions` ?

Reply:
:+1: